src/python_optimiser_interface.cpp

```diff
@@ -871,6 +871,70 @@ extern "C"
     return PyBool_FromLong(optimiser->refine_partition);
   }
 
+  PyObject* _Optimiser_set_max_comm_size(PyObject *self, PyObject *args, PyObject *keywds)
+  {
+    PyObject* py_optimiser = NULL;
+    unsigned int max_comm_size = 0;
+    static char* kwlist[] = {"optimiser", "max_comm_size", NULL};
+
+    #ifdef DEBUG
+      cerr << "Parsing arguments..." << endl;
+    #endif
+
+    if (!PyArg_ParseTupleAndKeywords(args, keywds, "OI", kwlist,
```

Comment:
For reading a `size_t` object, you would need the specifier `n`.

Reply:
Done.